mapper/pojo-base/src/main/java/org/hibernate/search/mapper/pojo/mapping/definition/programmatic/PropertyMappingDocumentIdOptionsStep.java

```diff
@@ -41,10 +44,28 @@ default PropertyMappingDocumentIdOptionsStep identifierBridge(IdentifierBridge<?
 	}
 
 	/**
+	 * Define an identifier binder, responsible for creating a bridge.
+	 * To pass some parameters to the bridge,
+	 * use the method {@link #identifierBinder(IdentifierBinder, Map)} instead.
+	 *
+	 * @param binder A {@link IdentifierBinder} responsible for creating a bridge.
+	 * @return {@code this}, for method chaining.
+	 * @see IdentifierBinder
+	 */
+	default PropertyMappingDocumentIdOptionsStep identifierBinder(IdentifierBinder binder) {
+		return identifierBinder( binder, Collections.emptyMap() );
+	}
+
+	/**
+	 * Define an identifier binder, responsible for creating a bridge.
+	 * With this method it is possible to pass a set of parameters to the binder,
+	 * so that they can be used by the bridge.
```

Comment:
```suggestion
	 * With this method it is possible to pass a set of parameters to the binder.
```


Parameters don't have to be used by the bridge, necessarily. They can just control mapping options (is this field sortable, ...).